packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -143,7 +149,12 @@ export class DocumentDeltaConnection
 
         this.submitManager = new BatchManager<IDocumentMessage[]>(
             (submitType, work) => {
-                this.socket.emit(submitType, this.clientId, work);
+                // Although the implementation here disconnects the socket and does not reuse it, other subclasses
+                // (e.g. OdspDocumentDeltaConnection) may reuse the socket.  In these cases, we need to avoid emitting
+                // on the still-live socket.
+                if (!this.closed) {
+                    this.socket.emit(submitType, this.clientId, work);
+                }
```

Comment:
do we have the ability to log something here. would be good to know when this happening

Reply:
I'm not sure how to instrument correctly here, but also I think it might be non-actionable.  There's no external exposure from `BatchManager` that it has pending work, so it's not possible to time the `disconnect()` call to avoid preempting the pending work.  Any normal call to `disconnect()` risks hitting this scenario.

Reply:
Agree, without some logging this could obfuscate the issue. Given the low-ish number of hits we're seeing in the other telemetry, we wouldn't be able to differentiate between if this resolves some of the hits or none of the hits for example.

Reply:
Per conversation with Vlad, Tony, and Helio, we will not add instrumentation here since it's generally expected that some percentage of disconnect calls will hit this scenario (whenever the `BatchManager` still has outstanding items in its queue), so the data would not be particularly actionable.